server/api/statistics/statistics.controller.js

```diff
@@ -59,6 +54,123 @@ const getSummary = async (req, res) => {
   }
 }
 
+const getLocations = async (req, res) => {
+  const { orgId, timeframe } = req.params
+
+  // authentication
```

Comment:
You use this authentication and authorisation pattern fairly frequently in your api endpoints and may want to factor it out.